projects/components/src/open-in-new-tab/open-in-new-tab.component.ts

```diff
@@ -8,11 +8,9 @@ import { IconSize } from '../icon/icon-size';
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <div *ngIf="this.paramsOrUrl" class="open-in-new-tab" htTooltip="Open in a new tab">
-      <ht-event-blocker event="click">
-        <ht-link [paramsOrUrl]="this.paramsOrUrl" ariaLabel="Open in a new tab">
-          <ht-icon icon="${IconType.OpenInNewTab}" [size]="this.iconSize"></ht-icon>
-        </ht-link>
-      </ht-event-blocker>
+      <ht-link [paramsOrUrl]="this.paramsOrUrl" ariaLabel="Open in a new tab">
```

Comment:
we need to add the event blocker back in here since this component could be used within a cell of a table or card.

Reply:
It is not working with the event blocker. When I removed it, it started working. My guess would be since we're using anchor tag directly with href. In the internal one, we're using `routerLink` instead, so angular was taking care of the navigation. Should we also use a click method for this which can do the navigation?

Reply:
Now I think about it that means, every link with event blocker wouldn't work for external navigation

Reply:
Updated the PR to fix this.